pkgs/by-name/se/seabird/package.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, buildGo122Module
+, copyDesktopItems
+, fetchFromGitHub
+, pkg-config
+, wrapGAppsHook4
+, glib
+, gobject-introspection
+, gtk4
+, gtksourceview5
+, libadwaita
+, libxml2
+, vte-gtk4
+}:
+
+buildGo122Module rec {
+  pname = "seabird";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "getseabird";
+    repo = "seabird";
+    rev = "v${version}";
+    hash = "sha256-wEZS5wOiSaCJ5jEx5tfp6QxY0SgwP5e7MV+iqekHmPY=";
+  };
+
+  vendorHash = "sha256-z9l6g5NkAErRQo8oiqwKG9ssm8K2S+eSZBD0w4kO3kc=";
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    glib
```

Comment:
I don' think glib is used

Reply:
I added it because of what `ldd` output:

```
$ ldd /nix/store/bnbnq9sgzpnimwj3k1vixjdwh9a8l2hg-seabird-0.2.1/bin/seabird
	linux-vdso.so.1 (0x00007ffdb2bc2000)
	libc.so.6 => /nix/store/8mc30d49ghc8m5z96yz39srlhg5s9sjj-glibc-2.38-44/lib/libc.so.6 (0x00007f0108c59000)
	/nix/store/8mc30d49ghc8m5z96yz39srlhg5s9sjj-glibc-2.38-44/lib/ld-linux-x86-64.so.2 => /nix/store/8mc30d49ghc8m5z96yz39srlhg5s9sjj-glibc-2.38-44/lib64/ld-linux-x86-64.so.2 (0x00007f0108e44000)

$ nix shell 'nixpkgs#nix-index' -c nix-locate libc.so.6 | grep -v '('
remarkable-toolchain.out                              0 s /nix/store/d69721qdx5y5iysfg5q1mnc5c4bh66ni-remarkable-toolchain-3.1.2/sysroots/cortexa9hf-neon-remarkable-linux-gnueabi/lib/libc.so.6
remarkable-toolchain.out                              0 s /nix/store/d69721qdx5y5iysfg5q1mnc5c4bh66ni-remarkable-toolchain-3.1.2/sysroots/x86_64-codexsdk-linux/lib/libc.so.6
remarkable2-toolchain.out                             0 s /nix/store/y59nachj812qs309ikz8gw6dx78m4gvv-remarkable2-toolchain-3.1.2/sysroots/cortexa7hf-neon-remarkable-linux-gnueabi/lib/libc.so.6
remarkable2-toolchain.out                             0 s /nix/store/y59nachj812qs309ikz8gw6dx78m4gvv-remarkable2-toolchain-3.1.2/sysroots/x86_64-codexsdk-linux/lib/libc.so.6
glibc_memusage.debug                                  0 s /nix/store/km9ggz0h354f3vldgzvgjjyiblvw8hn5-glibc-gd-2.37-45-debug/lib/debug/libc.so.6
glibc_memusage.out                            2,252,736 x /nix/store/y51mzms9hyaz3khsmahvwp1i5n9jfpp3-glibc-gd-2.37-45/lib/libc.so.6
glibc.out                                     2,252,736 x /nix/store/whypqfa83z4bsn43n4byvmw80n4mg3r8-glibc-2.37-45/lib/libc.so.6
glibc_multi.out                                       0 s /nix/store/pq23w2wmy4kzndyv06c1a6k0mkpqhff4-glibc-multi-2.37-45/lib/libc.so.6
glibc.debug                                           0 s /nix/store/fd22xxhwvz51drm2i3yzzbyj1b363mgp-glibc-2.37-45-debug/lib/debug/libc.so.6
```

I’m still confused where to put dependencies, maybe it should go in `buildInputs` instead?

Reply:
`glib` and `glibc` are very different, though.

Reply:
:facepalm: What a dummy… Thanks for pointing this out!